README.md

```diff
@@ -70,9 +70,8 @@ Cicero currently only ships a Nix evaluator but others are planned.
 For actions written in Nix, Cicero provides a standard library of functions
```

Comment:
Beginning of this file says:

> Think of it like an if-this-then-that machine on HashiCorp Nomad.
Cicero’s actions are flexible enough to build Continuous Integration (CI) and
Continuous Delivery (CD) pipelines. It offers a rich Web UI as well as a CLI
tool for developers to query and inspect actions and their runs,
as well as the action output.

This description spawned wired and complex image in my head and caused many hours of confusion and fear. IMHO it would be easier to understand if we mention well know tools/terminology used by programmers (as they are main audience):
1) CI/CD tools like Jenkins, TeamCity
2) REST-like API
3) tech stack

I would say something like:

Cicero can be seen as CI/CD tool like Jenkins. It has web interface that allows creating, inspecting and manage build/deploy jobs called actions. Actions execution are called runs.

You can interact with it sending POST requests with JSON.

Pipelines are defined using nix and HashiCorp tech stack (Nomad, Vault).

Reply:
Sounds good. I would change the last part a little:
Actions are defined in the Nix expression language which is extended by Cicero specific libraries.
Action Runs are converted into Jobs which get deployed to Nomad.
We also use Vault from the Hashicorp tech stack to enable authentication.

Reply:
I made sure that the handbook and readme introduction is the same and a bit more in detail.